### ISSUE TITLE
aliasByTags not working correctly

### ISSUE THREAD

**Issue by Hipska** [go-graphite/carbonapi#422]
Hi, it seems that `carbonapi` (0.12.5) does not support tag queries very well when there is also some computation.

The following query displays the series as `perSecond(interface_in_octets` and `perSecond(interface_out_octets`:
```graphite
aliasByTags(perSecond(seriesByTag('hostname=$host', 'ifName=~${interface:regex}', 'name=~interface_(in|out)_octets')), 'name')
```
While the following query works correct (but does not show correct data):
```graphite
aliasByTags(seriesByTag('hostname=$host', 'ifName=~${interface:regex}', 'name=~interface_(in|out)_octets'), 'name')
```

It seems similar as #390 but it does not crash in this case, so I'm making a new issue.

**Comment by Civil** [comment]
And what are the values for $host and $interface?

Ideally it would be great if you'll be able to give some example names and tags for the series that matches this 'seriesByTag' query.

Also you've said that just aliasByTags(seriesByTags(...)) works, but data is incorrect - what do you mean by that (except for perSecond function not being applied)?

**Comment by Hipska** [comment]
Okay then assume `gateway1` for host and `GigabitEthernet0/0/0` for interface.

The data is wrong indeed just because the `perSecond` function is not applied. I'm just talking about the naming of the resulting series.

**Comment by Civil** [comment]
Fixed in current master

**Comment by Hipska** [comment]
Great thanks, any idea when a new version will be released?

**Comment by Civil** [comment]
I want to fix #417 before tagging a new release. However I might tag 0.12.6 with only this fix and panic fix applied. Not sure though if it's worth it.
